source/includes/_result.md

```diff
@@ -3,124 +3,161 @@
+        }
+      },
+      {
+        "id": "tnEKYuedAjPvBC6EFwAL",
+        "text": "$125,000 to $249,999",
+        "question": {
+          "data": {
+            "id": "znEPYuedAjPva",
+            "type": "question",
+            "attributes": {
+              "text": "What is the total value of your liquid assets...",
+              "description": "Generally, the greater the liquid assets you have, the more capacity you have to take on risk...",
+            },
             "links": {
-              "self": "https://..."
+              "self": "http://app.sixpark.com.au/api/v1/questionnaire/questions/tnEKYuedAjPvBC6EFwAL",
+              "related": "http://app.sixpark.com.au/api/v1/questionnaire/questions"
             }
           }
-        },
-        "..."
-      ]
-    },
-    "..."
-  ],
+        }
+      },
+      "..."
+    ],
+    "conflicts": [
+      {
+        "text": "It is unusual to seek high-risk investments when...",
+        "answers": [
+          {
+            "id": "EGJkOzOij5PUhMZsjUno",
+            "text": "Mostly concerned with potential...",
+            "question": {
+              "data": {
+                "id": "tnEKYuedAjPvB",
+                "type": "question",
+                "attributes": {
+                  "text": "Which statement best describes your primary concern...",
+                  "description": "This question helps assess your desire for risk..."
+                },
+                "links": {
+                  "self": "http://app.sixpark.com.au/api/v1/questionnaire/questions/EGJkOzOij5PUhMZsjUno",
+                  "related": "http://app.sixpark.com.au/api/v1/questionnaire/questions"
+                }
+              }
+            }
+          },
+          {
+            "id": "QrVhgp9VAw2LOMC7kMPh",
+            "text": "I want to play it safe and protect...",
+            "question": {
+              "data": {
+                "id": "tnEKYuedAjPvB",
+                "type": "question",
+                "attributes": {
+                  "text": "Which statement BEST describes your current...",
+                  "description": "This question helps assess your desire for risk..."
+                },
+                "links": {
+                  "self": "http://app.sixpark.com.au/api/v1/questionnaire/questions/EGJkOzOij5PUhMZsjUno",
+                  "related": "http://app.sixpark.com.au/api/v1/questionnaire/questions"
+                }
+              }
+            },
+            "...",
+          },
+          "...",
+        ],
+      },
+    ],
+  },
   "links": {
-    "sign_up": "https://...",
-    "self": "https://..."
+    "self": "http://app.sixpark.com.au/api/v1/questionnaire/results/QUc2RHBMa1V3TENDTnlaUmhIRWksZE56d1ZhTlNCL3hoWFRya2ZvQjEsSVJZNU9GeHNlSytIWmZVZUNwUXgsR3lEYzBxYnR0dWpSdmdDWkcvTjIsRUdKa096T2lqNVBVaE1ac2pVbm8sSkFIQktpUGlqc08wNGVhMmxxVk0seC9ScXlZbVFVWnpPSU5weVNWY3EsZkp1KzRSU2tueDM0eklPSXE3TW8sdHhxM0pId2VXR3dyTkRjS2VERXosUVJGZ3pUVVFMc29Dd0pRRkRwUU0="
```

Comment:
`https` and it may be worth using the same id as in the example `QUc2RHBMa1V3TENDTnlaUmhIRWksdG5FS1l1ZWRBalB2QkM2RUZ3QUwK` to be consistent.

Reply:
ahh yep nice catch